test/fixtures.yml

```diff
@@ -393,8 +393,9 @@ instapay:
   login: TEST0
   password:
 
-# Working credentials, no need to replace
+# login: merchant id
```

Comment:
why the change in comment?

Reply:
I think there is no exact function for this change, I will revert it.